src/xrpld/app/tx/detail/DeleteAccount.cpp

```diff
@@ -51,6 +52,10 @@ DeleteAccount::preflight(PreflightContext const& ctx)
         // An account cannot be deleted and give itself the resulting XRP.
         return temDST_IS_SRC;
 
+    auto const err = DepositPreauth::preflightCheckCredentials(ctx.tx, ctx.j);
```

Comment:
This has to be amendment guarded as well. And other transactors too.

Reply:
Checks are inside functions.